### ISSUE TITLE
Feature request: a recommended shared config?

### ISSUE THREAD

**Issue by JayAndCatchFire** [amilajack/eslint-plugin-flowtype-errors#117]
Assuming https://github.com/amilajack/eslint-plugin-flowtype-errors/pull/116 gets merged, there'll now be two rules (with kinda obvious defaults).  `show-errors` and `show-warnings`.  How does a `recommended` shared config sound that sets these?
```js
"show-errors": 2,
"show-warnings": 1
```

**Comment by amilajack** [comment]
That sounds good 👍

**Comment by JayAndCatchFire** [comment]
I'll work on a PR for this once #116 gets merged.
